### ISSUE TITLE
SMTP - Sign-in attempt prevented

### ISSUE THREAD

**Issue by mariodian** [OpenBazaar/OpenBazaar-Server#395]
I tried to use gmail's smtp for notifications and I got this email from google:

![screen shot 2016-05-11 at 8 23 16 pm](https://cloud.githubusercontent.com/assets/3636406/15180870/a2c312d6-17b6-11e6-9094-dd304a4f3965.png)

Perhaps certain headers need to be used so it doesn't look suspicious? This can't be fixed by clicking "review your devices now" button, because OB isn't seen there.

**Comment by hoffmabc** [comment]
This is because unsafe apps that don't use 2fa throw warnings. You have to enable unsafe apps. I would recommend creating a throw away account as a relay for now. Keep in mind this is less secure.

**Comment by lunokhod** [comment]
Notifications work fine for me with gmail. Are you using 2FA with an application specific password?

**Comment by mariodian** [comment]
Yes I use 2FA but I think not for this account. It's a secondary one.

**Comment by inertia186** [comment]
I had unsafe apps enabled on the Google account I tried, still got the "review your devices now" message.  This is, of course, a Google-created problem.  But from a support standpoint, it's going to be a lot of ongoing trouble for OpenBazaar.

In fact, Google makes it hard.  There's multiple places where "less secure apps" can be enabled.  The one that worked is here (scroll down to "Allow less secure apps"):

https://myaccount.google.com/u/2/security#connectedapps
